src/incubator/TextField/CharCounter.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {useContext} from 'react';
+import {TextStyle, StyleSheet} from 'react-native';
+import _ from 'lodash';
+import Text from '../../components/text';
+import FieldContext from './FieldContext';
+
+export interface CharCounterProps {
+  showCharCounter?: boolean;
+  maxLength: number;
```

Comment:
Do you want to revert the change here and keep it optional?

Reply:
Changed it back to optional
and moved the check for `maxLength` to CharCounter